mcts.py

```diff
@@ -35,6 +36,9 @@
 def D_NOISE_ALPHA(): return 0.03 * 19 / go.N
 
 
+# Progressive widening
```

Comment:
Can you document here what progressive widening is and why it's useful? Part of the goal of the project is to be understandable to ml newbies like myself (although obviously, we have a long way to go).

Reply:
Progressive widening is just a name I made up. This pull is more of a way to start a conversation about what to do, I'm not very comfortable with this as is but the results are quite good. When things become more clear I'll add some notes or a link to something. For now you can follow the long thread here, or look at my most recent summary post:
https://github.com/gcp/leela-zero/issues/696#issuecomment-362687915